### PR TITLE
Changes for all previous observations in Skywater

### PR DESCRIPTION
**Pull request by microSharjeel**
Hi
We can check this for all types of mems for both techs
to see if there are any errors

regards

### CODE REVIEW

**Comment by jjts** [comment]
added Joao as reviewer

File: software/python/memakerwrap.py [hunk 1]
```diff
@@ -166,114 +240,256 @@ def instWires (type, async, be) :
...
+		    print ""
+		    print "    .CS(en),"
+		    #print "    .OE(oe)," it is not req in skywater130
+        elif type == "sprom":
+		    for i in range(bits):
+		        print "    .DO"+str(i)+"(r_data["+str(i)+"]),"
+		    print "    .CS(r_en),"
+		    print "    .OE(oe),"
         print ""
-        if bytes > 1:
```
Comment (JDLopes) [inline software/python/memakerwrap.py:235]: Why did you removed these lines?

**Review by JDLopes — COMMENTED** [review]
Please, check the comments.

File: software/python/memakerwrap.py (hunk 1 above)
Comment (JDLopes) [reply software/python/memakerwrap.py:235]: Forget about the question. From the way GitHub shows the differences, it seemed you had removed these lines. My fault!

**Review by JDLopes — APPROVED** [review]
It's ok! Good work!

**Pull request merged**